src/main/java/com/zerobase/moviereservation/auth/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.zerobase.moviereservation.auth;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
+  public static final String TOKEN_HEADER = "Authorization";
+  public static final String TOKEN_PREFIX = "Bearer ";
+  private final TokenProvider tokenProvider;
+
+  @Override
+  protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
+      FilterChain filterChain) throws ServletException, IOException {
+
+    String token = this.resolveTokenFromRequest(request);
+
+    if (StringUtils.hasText(token) && this.tokenProvider.validateToken(token)) {
+      Authentication auth = this.tokenProvider.getAuthentication(token);
+      SecurityContextHolder.getContext().setAuthentication(auth);
+
+      log.info(String.format("[%s] -> %s", this.tokenProvider.getUsername(token),
+          request.getRequestURI()));
```

Comment:
logger에서는 String.format을 쓰지 않아도 됩니다. 
```suggestion
      log.info("[{}] -> {}", this.tokenProvider.getUsername(token),
          request.getRequestURI());
```